roborock/devices/traits/v1/__init__.py

```diff
@@ -103,7 +154,7 @@ def __init__(
                 # We exclude optional features and them via discover_features
                 if (union_args := get_args(item.type)) is None or len(union_args) > 0:
                     continue
-                _LOGGER.debug("Initializing trait %s", item.name)
+                _LOGGER.debug("Trait '%s' is supported, initializing", item.name)
```

Comment:
Note: Updated the debug logging to be more consistent and readable across the different cases